rpcs3/headless_application.cpp

```diff
@@ -137,6 +137,7 @@ void headless_application::InitializeCallbacks()
 	callbacks.on_resume = []() {};
 	callbacks.on_stop   = []() {};
 	callbacks.on_ready  = []() {};
+	callbacks.on_emulation_stop_no_response = [](std::shared_ptr<atomic_t<bool>> closed_successfully) { if (!*closed_successfully) std::abort(); };
```

Comment:
missing check for closed_successfully ptr

Reply:
Fixed.